src/pages/Reservation/ReservAside/ReservAside.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export default function ReservAside() {
+  return (
+    <ReservAsideWrapper>
```

Comment:
```suggestion
    <Container>
```
저는 가장 바깥쪽 요소는 Container로 통일하는 편입니다